packages/coinstac-pipeline/src/pipeline-manager.js

```diff
@@ -192,12 +255,64 @@ module.exports = {
         // TODO: step check?
         if (!data.error && activePipelines[data.runId]) {
           activePipelines[data.runId].state = 'recieved central node data';
-          activePipelines[data.runId].remote.resolve(data.output);
+          if (data.files) {
+            // we've already recieved the files
+            if (activePipelines[data.runId].files
+              && activePipelines[data.runId].files.recieved
```

Comment:
Typo on `recieved`. Maybe you're not accessing the property you want?

Reply:
I dont see the typo?

Reply:
jk, errything is spelled wrong.....